jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EjbSchedulerService.java

```diff
@@ -78,7 +88,21 @@ public JobHandle scheduleJob(Job job, JobContext ctx, Trigger trigger) {
 
     @Override
     public boolean removeJob(JobHandle jobHandle) {
-        return scheduler.removeJob(jobHandle);
+        JtaTransactionManager tm = (JtaTransactionManager) TransactionManagerFactory.get().newTransactionManager();
```

Comment:
Two questions:
- Does it have to be JtaTransactionManager or could it be valid to use the parent TransactionManager?
- In the case of no transactional EJB Timers ("org.jbpm.ejb.timer.tx"=false), is this still valid?

Reply:
yes.... 
1) it is always jta in case of ejb
2) and the ejb timer is always executed within CMT boundaries. the problem was CMT -> BMT (ejb scheduler) this pauses the current tx making outside the curren tx.